.pre-commit-config.yaml

```
@@ -40,3 +40,8 @@ repos:
     hooks:
       - id: docformatter
         args: ["--in-place", "--wrap-descriptions", "79"]
+  - repo: https://github.com/open-mmlab/pre-commit-hooks
+    rev: v0.2.0
+    hooks:
+    -   id: check-copyright
+        args: [ "mmrazor", "tests", "tools"]
```

Maybe we should also add `check-algo-readme`
```suggestion
  - repo: https://github.com/open-mmlab/pre-commit-hooks
    rev: v0.2.0
    hooks:
    -   id: check-algo-readme
    -   id: check-copyright
        args: [ "mmrazor", "tests", "tools"]
```